tensorflow/tools/graph_transforms/fold_old_batch_norms.cc

```diff
@@ -159,6 +159,7 @@ Status FuseScaleOffsetToConvWeights(const std::vector<float>& scale_values,
   NodeDef bias_add_node;
   bias_add_node.set_op("BiasAdd");
   bias_add_node.set_name(conv_output_name);
+  CopyNodeAttr(conv_node, "data_format", "data_format", &bias_add_node);
```

Comment:
This would fail the whole program if the conv_node doesn't have "data_format" attribute. You probably need to first verify it has the attribute "data_format" before calling CopyNodeAttr?

Reply:
I don't see any checks like this in any other part of the code, please could you direct me to some.

Also, since it is now part of the api in `tf.nn.conv2d` does it not get this attribute regardless?  - I'm not sure but I thought that it would fallback to the default.

Reply:
This is just an extra safety check for legacy tf graphs. You are right, If the original graph is created with tf.nn.conv2d, then this check is not needed; but I am not whether there are any legacy tf graphs that do not contain this attr.